overrides/android-override.json

```diff
@@ -577,6 +577,35 @@
         "clientContentFeatures": {
             "state": "enabled",
             "exceptions": [ ]
+        },
+        "mediaPlaybackRequiresUserGesture": {
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "clearmemorysolution.com"
+                },
+                {
+                    "domain": "id.onlyfans.com"
+                },
+                {
+                    "domain": "naturemetabolism.com"
+                },
+                {
+                    "domain": "upperbodystretch.com"
+                },
+                {
+                    "domain": "app.trunarrative.cloud"
+                },
+                {
+                    "domain": "cloudapi.trustid.co.uk"
+                },
+                {
+                    "domain": "facialwebcapture.cloud.itau.com.br"
+                },
+                {
+                    "domain": "tinder.com"
```

Comment:
Added one more domain. See https://app.asana.com/0/0/1206420805430540/1206500180540104/f and https://app.asana.com/0/0/1206420805430540/1206495276207664/f.